float64 X_e 
float64 Y_e 
float64 Z_e 
float64 yaw_e 
float64 Vx_e 
float64 Vy_e 
float64 Vz_e
